data/sql/updates/pending_db_world/skeletalhorrors.sql

```diff
@@ -1,4 +1,4 @@
-UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 202;
+
```

Comment:
Remove the extra line on top or add a comment saying the changes

example: -- From Cast to Near Players (2) use Terrify